doc/man3/CMS_EncryptedData_set1_key.pod

```diff
@@ -0,0 +1,43 @@
+=pod
+
+=head1 NAME
+
+CMS_EncryptedData_set1_key - Sets the cipher and key for
```

Comment:
Please remove this symbol from util/missingcrypto.txt

Reply:
Done. Next Push will have it removed from the file missingcrypto.txt.